ajax/libs/jquery-advanced-news-ticker/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "jquery-advanced-news-ticker",
+  "filename": "newsTicker.min.js",
+  "description": "A powerful, flexible and animated vertical news ticker plugin for JQuery.",
+  "keywords": [
+    "news",
+    "ticker",
+    "newsticker",
+    "animation",
+    "scroll",
+    "scroller",
+    "slide",
+    "slider",
+    "animated",
+    "callback",
+    "jquery",
+    "vertical"
+  ],
+  "version": "1.0.1",
+  "author": {
+    "name": "risq",
+    "url": "https://github.com/risq"
+  },
+  "license": "GPL-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/risq/jquery-advanced-news-ticker.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/risq/jquery-advanced-news-ticker.git",
+    "fileMap": [
+      {
+        "basePath": "js",
+        "files": [
+          "newsTicker*"
```

Comment:
The latest master branch of [jquery-advanced-news-ticker](https://github.com/risq/jquery-advanced-news-ticker.git) seems rename the file to `jquery.newsTicker.js` and `jquery.newsTicker.min.js`. Let's use `"*newsTicker*"` here.